.travis.yml

```diff
@@ -109,13 +108,13 @@ jobs:
       name: linux
       os: linux
       dist: xenial
-
       script:
         - export APP_VERSION=$(git describe --abbrev=0 --tags | tr -d '^v')
         - $TRAVIS_BUILD_DIR/scripts/ci_create_packages.sh
+        - $TRAVIS_BUILD_DIR/scripts/ci_build_generate_secret.sh
 
     - stage: Build and Upload
-      if: type = push AND tag IS present
+      if: type = push AND tag IS present AND tag =~ /^v[0-9]+\.[0-9]+\.[0-9]+$/
       name: linux
```

Comment:
this makes it so that if we push tags like v1.0.1-preview then that wont actually be a release. only tags that look like v1.0.1  (just v and numbers) will be triggering a publish